cmd/bosun/database/state_data.go

```diff
@@ -150,6 +192,45 @@ func (d *dataAccess) GetAllIncidents(ak models.AlertKey) ([]*models.IncidentStat
 	return d.incidentMultiGet(conn, ids)
 }
 
+func (d *dataAccess) getAllIncidentIds() ([]int64, error) {
```

Comment:
Why are these two functions both needed? Didn't you say you stopped using on or the other?